charts/posthog/templates/_snippet-removed-vars-errors.tpl

```diff
@@ -0,0 +1,24 @@
+{{/* Checks whether removed variables are in use. */}}
```

Comment:
Should we simply add this to `_helpers.tpl` ?

Reply:
I think it's more likely it will get lost in the noise and never updated if it was in that file.